docs/source/running.rst

```diff
@@ -181,6 +181,24 @@ is simple to run from the command, e.g:
 .. note::
    Uvicorn is as easy to install as ``pip install uvicorn``
 
+
+Django
+^^^^^^
+
+`Django`_ is a Python web framework that encourages rapid development and clean, pragmatic design.  Assuming
+a Django install/enabled application:
```

Comment:
I would add this part coherently:

```console
HTTP request <--> Django (pygeoapi/django_app.py) <--> pygeoapi API (pygeoapi/api.py)

The Django server can be run as follows:

pygeoapi serve --django
```

Reply:
I added the workflow to the `examples/django/sample_project/README.md` and added the rest to `docs/source/running.rst`.